packages/esm-ward-app/src/index.ts

```diff
@@ -34,8 +36,16 @@ export const wardPatientWorkspace = getAsyncLifecycle(
   options,
 );
 
+export const wardPatientNotesWorkspace = getAsyncLifecycle(
+  () => import('./ward-workspace/ward-patient-notes/notes.workspace'),
+  options,
+);
+
 export const wardPatientActionButtonExtension = getSyncLifecycle(WardPatientActionButton, options);
 
+export const wardPatientNotesActionButtonExtension = getSyncLifecycle(WardPatientNotesActionButton, options);
+export const wardPatientNotesFormExtension = getSyncLifecycle(PatientNotesForm, options);
```

Comment:
Why are we fetching these synchronously?

Extensions which should render directly on the UI on the first load, say Ward View, should be imported synchronously to reduce the API calls to fetch data, else all other things should be imported asynchronously.

Reply:
Few examples will be:
1. Left nav items on Home page/ Patient chart
2. Items on the Primary Navigation